CodePush.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name                = 'CodePush'
-  s.version             = '1.10.1-beta'
```

Comment:
thank you @lostintangent ... I've updated this to append '-beta' to the tag. now the tags will be maintained but the version that shows up in the Info.plist will be in the proper format.